src/Forms/FieldList.php

```diff
@@ -497,7 +472,7 @@ public function fieldByName($name)
      * @param string $name The name of the field to return
      * @return FormField|null
```

Comment:
```suggestion
```
I've pointed out a few of these where the PHPDoc doesn't give any useful context that isn't implied by the method signature, but please take a quick look for any I might have missed.

Reply:
Removed